src/CMakeLists.txt

```diff
@@ -2233,6 +2236,13 @@ ELSE (NOT WIN32)
     ENDIF(VISIT_MAKE_NSIS_INSTALLER)
 ENDIF (NOT WIN32)
 
+#
+# Build the sphinx documentation.
+#
+execute_process(
```

Comment:
By "dependencies", do you mean the libraries (python3, sphinx), or do you mean the actual building of the documentation?

Reply:
I guess I am wondering whether this new code in top level `CMakeLists.txt` forces us to *require* python3 and Sphinx in order to simply create `Makefiles` and/or do a `make`. I dunno but it looks like this is UNconditionally included/executed in `CMakeLists.txt` If if does, I think it should be re-worked to avoid it being *required* but still *optional*.

Reply:
I've updated the CMake process to add a custom target instead of building the manuals by default. 
In order to build the manuals, you'll now need to invoke the command "make manuals" from src. This change doesn't, however, affect whether or not the libraries are built. Those are still built by default. I can change that if we want.